### ISSUE TITLE
Behavior of dash in core->port mapping does not match documentation

### ISSUE THREAD

**Issue by DoctorSher** [pktgen/Pktgen-DPDK#208]
Hey Keith,

There is an example in the documentation to assign multiple cores to RX/TX processing on a particular queue using a dash. It is the following example:
```
[0-1].0, [2/4-5].1, ...       - cores 0-1 handle port 0 rx/tx,
                                cores 2,4,5 handle port 1 rx/tx
```
From the description, I would expect core 0 and core 1 to each be doing RX and TX processing for port 0. Using this format, I am running the following Pktgen command 
```
sudo -E ./pktgen -l 0-7 -n 4 -w 0000:0a:00.0 -w 0000:0a:00.1 -- -P -m "[1-4].0, [5-7].1"
```
However, the output I receive from the Pktgen initialization does not suggest that cores 1 through 4 will each be doing RX/TX processing, but rather that core 1 will do RX/TX processing and cores 2 through 4 will be doing RX processing only. The relevant portion of the output is below:
```
  RX/TX processing lcore:   1 rx:  1 tx:  1
  RX processing lcore:   2 rx:  1 tx:  0
  RX processing lcore:   3 rx:  1 tx:  0
  RX processing lcore:   4 rx:  1 tx:  0
  RX/TX processing lcore:   5 rx:  1 tx:  1
  RX processing lcore:   6 rx:  1 tx:  0
  RX processing lcore:   7 rx:  1 tx:  0
```
If I change my command to the following:
```
sudo -E ./pktgen -l 0-7 -n 4 -w 0000:0a:00.0 -w 0000:0a:00.1 -- -P -m "[1-4:1-4].0, [5-7:5-7].1"
```
I get the output and behavior I expect, where cores 1 through 4 do RX/TX processing for port 0 and cores 5 through 7 do RX/TX processing for port 1.
```
  RX/TX processing lcore:   1 rx:  1 tx:  1
  RX/TX processing lcore:   2 rx:  1 tx:  1
  RX/TX processing lcore:   3 rx:  1 tx:  1
  RX/TX processing lcore:   4 rx:  1 tx:  1
  RX/TX processing lcore:   5 rx:  1 tx:  1
  RX/TX processing lcore:   6 rx:  1 tx:  1
  RX/TX processing lcore:   7 rx:  1 tx:  1
```
I'm not sure which if the documentation doesn't match the intention of the program, or if the program doesn't match the intention of the documentation. However, if I were in your shoes, changing the documentation is probably a lot easier! ;)

Cheers,
Adam

**Comment by pktgen** [comment]
Yes changing the docs would be easier. I will look into the code and see if I did not account for that setup or just a bug. Most likely changing docs is the way go.
